### PR TITLE
AAP-29218 - Convert managing authentication content

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR converts the content for the managing authentication modules from the Google draft to AsciiDoc.

https://docs.google.com/document/d/1mEYguv3V0WUUHBMc8tsgqlK8RB3ro8IZSlQmMuFfhww/edit#bookmark=kix.sll4ydwxafjs

### CODE REVIEW

File: downstream/modules/platform/proc-gw-edit-authenticator.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+= Editing an authenticator
+
+You can modify the settings of previously configured authenticators from the *Authentication* list view. 
+
+.Procedure
+
+. From the navigation panel, select {MenuAMAuthentication}. 
+. In the list view, you can either:
++
+.. Select the edit (pencil) icon next to authenticator you want to modify, or
```
Comment (ianf77) [inline downstream/modules/platform/proc-gw-edit-authenticator.adoc:14]: image:leftpencil.png[Edit,15,15] icon

File: downstream/modules/platform/proc-gw-searching-authenticator.adoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+[id="gw-searching-authenticator"]
+
+= Searching for an authenticator
+
+You can search for a previously configured authenticator from the Authentication list view.
+
+.Procedure
+
+. From the navigation panel, select {MenuAMAuthentication}. 
+. In the search bar, enter an appropriate keyword for the authentication method  you want to search for and click the arrow icon.
```
Comment (ianf77) [inline downstream/modules/platform/proc-gw-searching-authenticator.adoc:12]: If it's a > icon, there's one in the images directory

**Review by ianf77 — COMMENTED** [review]
Couple of comments

File: downstream/modules/platform/proc-gw-edit-authenticator.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-gw-edit-authenticator.adoc:14]: Fixed

File: downstream/modules/platform/proc-gw-searching-authenticator.adoc (hunk 2 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-gw-searching-authenticator.adoc:12]: It's actually a right arrow and not a > symbol so I'm going to leave it as is for now.

**Pull request merged**